_pages/en/contact-us.md

```diff
@@ -6,4 +6,15 @@ lang: en
 permalink: /contact-us.html
 ---
 
-{{ site.data.i18n.general.notices.feedback[page.lang]}}
+
+<!-- {{ site.data.i18n.general.notices.feedback[page.lang]}} -->
+<br/>
+All work found on this site are ideas and proposals from the IT Strategy team within ESDC. Content here does not represent the official stance of ESDC nor the Government of Canada at large.
+<br/>
+<br/>
+Got interested in our work or have new ideas? We would be happy to answer your questions and set up a meeting with you. Connect with us  through email.
```

Comment:
I'm assuming we will add a link for 'email'

Reply:
Also I would remove 'Got' from the start of this sentence. And perhaps re:word to: "Are you interested in our work, or have ideas to share? Or maybe you have questions or have found a typo on this site?  Feel free to contact any of us (link to About Us page) directly, or send an email our generic inbox (link), or engage with us on GitHub through Issues (link) or Pull Requests. Here is some information on how to contribute (link).